.gitignore

```diff
@@ -1,4 +1,3 @@
 build
 __pycache__
-dist
```

Comment:
I would keep this, because the .exe is not a source file. The exe should go into a release

Reply:
fair point, I've already fixed here [18a8dbf](https://github.com/sean-nicholas/scroll-windows-v-desktops/pull/10/commits/18a8dbf0313f701f27d753ffcd613885b9a9bae1)